start.go

```diff
@@ -282,16 +119,7 @@ func start(c *cli.Context) {
 	fmt.Fprintf(stderr, "SAM CLI if you update your AWS SAM template.\n")
 	fmt.Fprintf(stderr, "\n")
 
-	log.Fatal(http.ListenAndServe(c.String("host")+":"+c.String("port"), router))
-
-}
+	// Start the HTTP listener
+	log.Fatal(http.ListenAndServe(c.String("host")+":"+c.String("port"), mux.Router()))
```

Comment:
Agreed - the general advice with Go is the closer a variable is used to it's creation point, the more acceptable a short variable name is.

This `c` object is used all over start.go/invoke.go, and it makes the code harder to read.